From: Gerald Schaefer <gerald.schaefer@de.ibm.com>
Subject: s390/mm: add kmsg descriptions for 2GB hugepages
Patch-mainline: Never, kmsg in sles only
References: bsc#1025376, FATE#321651, LTC#146794

Summary:     s390/mm: add support for 2GB hugepages
Description: With this feature, the Linux hugetlbfs filesystem can use 2 GB
             hugepages. Access the hugepage memory either through the mmap()
             or SysV shared memory system calls. Using hugepage memory can
             accelerate memory accesses and reduce page table memory
             consumption. This feature is supported on zEC12 and later.

Upstream-Description:

             s390/mm: add kmsg descriptions for 2GB hugepages

Signed-off-by: Gerald Schaefer <gerald.schaefer@de.ibm.com>
Acked-by: John Jolly <jjolly@suse.com>
---
 Documentation/kmsg/s390/hugetlb |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

--- /dev/null
+++ b/Documentation/kmsg/s390/hugetlb
@@ -0,0 +1,14 @@
+/*?
+ * Text: "hugepagesz= specifies an unsupported page size %s\n"
+ * Severity: Error
+ * Parameter:
+ *   @1: size
+ * Description:
+ * The hugepagesz= kernel parameter specifies a huge page size
+ * that is not supported.
+ * User action:
+ * Specify "1M" for 1 MB huge pages. These are supported as of z10.
+ * Specify "2G" for 2 GB huge pages. These are supported as of zEC12
+ * and zBC12 machines.
+ */
+/*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
